joomla/joomla-cms
# Move JHttpFactory::getHttp() into try catch and add error handling

### Summary of Changes

Move any usage of  `JHttpFactory::getHttp()` into try catch and add error handling

### Testing Instructions

Review.

Confirm that you can still uses a feed.
Confirm that recaptcha still works.

### Expected result

everything still works

### Actual result

everything works but without error handling

### Documentation Changes Required

None.
